docs/development/project-development-environment.md

```diff
@@ -8,7 +8,7 @@
 | <a name='dependency_pg_config'>pg_config</a> (required by psycopg2) |  | <pre class="highlight highlight-source-shell">sudo apt install libpq-dev</pre> |
 | <a name='dependency_python3'>Python 3</a> | 3.4 | Would be pulled in by `python3-gdal` or <a href='#dependency_python3-dev'>`python3-dev`</a>, but you should install it explicitly with <pre class="highlight highlight-source-shell">sudo apt install python3</pre> |
-| <a name='dependency_requirements.txt'>various PyPI packages</a> | See [`requirements.txt`](/requirements.txt) and [`requirements-local.txt`](/requirements-local.txt) | Install in a Python 3 [virtual environment](#dependency_venv). Create one with e.g. <pre class="highlight highlight-source-shell">mkdir -p ~/.virtualenvs && \\<br />virtualenv ~/.virtualenvs/osmaxx -p python3</pre> activate it with <pre class="highlight highlight-source-shell">source ~/.virtualenvs/osmaxx/bin/activate</pre> Then, in the same shell session, use [`pip3`](#dependency_pip3) to install the packages: <pre class="highlight highlight-source-shell"># run from this repo's root dir<br />pip3 install -r requirements.txt \\<br />             -r requirements-local.txt</pre><hr>`virtualenvwrapper` users can perform all of the above in a single step: <pre class="highlight highlight-source-shell"># run from this repo's root dir<br />mkvirtualenv -r requirements.txt \\<br />             -r requirements-local.txt \\<br />             -a . -p $(which python3) osmaxx</pre>The `-a .` will also [associate the repo root as the project directory](http://virtualenvwrapper.readthedocs.org/en/latest/command_ref.html#mkvirtualenv) of the new Python 3 virtual environemnt. |
+| <a name='dependency_requirements.txt'>various PyPI packages</a> | See [`requirements.txt`](/requirements.txt), [`requirements-local.txt`](/requirements-local.txt) and [`requirements-all.txt`](/requirements-all.txt) | Install in a Python 3 [virtual environment](#dependency_venv). Create one with e.g. <pre class="highlight highlight-source-shell">mkdir -p ~/.virtualenvs && \\<br />virtualenv ~/.virtualenvs/osmaxx -p python3</pre> activate it with <pre class="highlight highlight-source-shell">source ~/.virtualenvs/osmaxx/bin/activate</pre> Then, in the same shell session, use [`pip3`](#dependency_pip3) to install the packages: <pre class="highlight highlight-source-shell"># run from this repo's root dir<br />pip3 install -r requirements-all.txt</pre><hr>`virtualenvwrapper` users can perform all of the above in a single step: <pre class="highlight highlight-source-shell"># run from this repo's root dir<br />mkvirtualenv -r requirements-all.txt \\<br />             -a . -p $(which python3) osmaxx</pre>The `-a .` will also [associate the repo root as the project directory](http://virtualenvwrapper.readthedocs.org/en/latest/command_ref.html#mkvirtualenv) of the new Python 3 virtual environemnt. |
```

Comment:
I think, either `requirements-mediator.txt` should be mentioned, too, **or** _only_ `requirements-all.txt` should be mentioned explicitly, e.g.
> See [`requirements-all.txt`](/requirements-all.txt) and the requirements files it refers to.